webpack.config.js

```diff
@@ -361,8 +360,9 @@ let appConfig = {
     ...(SHOULD_FORK_TS
       ? [
           new ForkTsCheckerWebpackPlugin({
-            eslint: TS_FORK_WITH_ESLINT,
```

Comment:
v7 will no longer support eslint. This has also become harder to use since now it requires you to tell it what files to watch.

I've just removed it for now

Reply:
See https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/releases/tag/v7.0.0-alpha.3